### PR TITLE
Support Ruby3

### PR DESCRIPTION
**Pull request by jmatsu**
The motivation is really clear. For Ruby3 support, this PR upgrade ActiveSupport and adds `webrick` to dependencies cuz Ruby 3 has dropped `webrick` from bundled gems.

### CODE REVIEW

File: lib/deploygate.rb [hunk 1]
```diff
@@ -12,6 +12,7 @@
 require "gem_update_checker"
 require "active_support/core_ext/time"
 require "active_support/core_ext/hash"
+require "active_support/core_ext/integer/time"
```
Comment (jmatsu) [inline lib/deploygate.rb:15]: active support has moved Numeric extension like `1.days` from `core_ext/time` to here.

File: lib/deploygate/api/v1/base.rb [hunk 2]
```diff
@@ -60,7 +60,7 @@ def headers
           extheaders = BASE_HEADERS.dup
 
           unless @token.nil?
-            extheaders.push(['AUTHORIZATION', @token])
+            extheaders.push(['AUTHORIZATION', "token #{@token}"])
```
Comment (jmatsu) [inline lib/deploygate/api/v1/base.rb:63]: The token header prefix is preferred.

File: lib/deploygate/commands/deploy/push.rb [hunk 3]
```diff
@@ -2,7 +2,6 @@ module DeployGate
   module Commands
     module Deploy
       class Push
-        BASE_URL = 'https://deploygate.com'
```
Comment (jmatsu) [inline lib/deploygate/commands/deploy/push.rb:5]: unused.

File: .github/workflows/pull_request.yml [hunk 4]
```diff
@@ -2,21 +2,131 @@ name: Run jobs on every PR
...
+      fail-fast: false
       matrix:
         ruby_version:
           - 2.6
           - 2.7
-          # - 3.0 # needs to upgrade fastlane and activesupport else
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
```
Comment (satsukies) [inline .github/workflows/pull_request.yml:21]: :+1:

File: deploygate.gemspec [hunk 5]
```diff
@@ -28,14 +30,15 @@ POST_INSTALL_MESSAGE
   spec.add_runtime_dependency 'highline', '~> 2.0'
   spec.add_runtime_dependency 'uuid', '~> 2.3'
   spec.add_runtime_dependency 'gem_update_checker', '~> 0.2'
-  spec.add_runtime_dependency 'activesupport', '~> 4.2'
-  spec.add_runtime_dependency 'i18n', '~> 0.7'
```
Comment (satsukies) [inline deploygate.gemspec:32]: I understand to delete this line because i18n gem was bundled with after Rails3.

**Review by satsukies — APPROVED** [review]
LGTM 👍

**Comment by jmatsu** [comment]
Thanks~

**Pull request merged**